WORKSPACE

```diff
@@ -271,3 +270,90 @@ gcc_register_toolchain(
     name = "gcc_toolchain_x86_64",
     target_arch = ARCHS.x86_64,
 )
+
+# Rust support
+http_archive(
+    name = "rules_rust",
+    integrity = "sha256-ww398ehv1QZQp26mRbOkXy8AZnsGGHpoXpVU4WfKl+4=",
+    urls = ["https://github.com/bazelbuild/rules_rust/releases/download/0.40.0/rules_rust-v0.40.0.tar.gz"],
+)
+
+load("@rules_rust//rust:repositories.bzl", "rules_rust_dependencies", "rust_register_toolchains")
+
+rules_rust_dependencies()
+
+rust_register_toolchains(
+    edition = "2021",
+    versions = [
+        # TODO(b/309612743) Can we automatically sync this with flake.nix?
```

Comment:
I think the TODO format is now different -- see http://go/todo-style . Could you let me know whether the linter step in CI caught this or not, so I can fix it if necessary?

Reply:
Sorry just seen this - CI did not complain. Maybe internally it will. But AFAIU both old and new formats are supported? But I agree to go with new format. I'll patch.